src/Microsoft.DocAsCode.Metadata.ManagedReference/Resolvers/CopyInheritedDocumentation.cs

```diff
@@ -139,7 +139,7 @@ private static void Copy(MetadataItem dest, string srcName, ResolverContext cont
             if (src == null && !context.Members.TryGetValue(srcName, out src))
             {
                 // Try to resolve any templated references before giving up
-                if (!context.References.TryGetValue(srcName, out var referenceItem) || !context.Members.TryGetValue(referenceItem.Definition, out src))
+                if (!context.References.TryGetValue(srcName, out var referenceItem) || referenceItem.Definition == null || !context.Members.TryGetValue(referenceItem.Definition, out src))
```

Comment:
I'd like to generate a warning here telling about the issue so the developer can fix their inheritdoc usage.
Can we create warnings here?

Reply:
yes, `Logger` is a static class that you can use to create warnings.